infinispan-client/src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 
 # Auth info
 quarkus.infinispan-client.auth-realm=default
-quarkus.infinispan-client.auth-username=qe
-quarkus.infinispan-client.auth-password=qe
+quarkus.infinispan-client.username=qe
```

Comment:
It looks like one of these changes broke the CI.

Reply:
well, I didn't run it as they should be one to one. I'll debug it.

Reply:
I got it working locally. Should be fixed now, let see.